WolvenKit.App/ViewModels/Shell/AppViewModel.cs

```diff
@@ -207,6 +207,14 @@ private bool TryLoadingArguments()
                 return false;
             }
 
+            if (args[1] == "--node_editor_enabled")
```

Comment:
Btw, this is a good idea, but in the long run I think it would make sense to move such checks to an appsettings or ini file or something - because the node editor will at some point just be globally enabled. The only reason I disabled it was to get it merged to mainline but hide it from users until we enable it on the nightly.

Reply:
I figured this was the reason, but wanted to give an option just in case. I could try and make such a change if you wanted? also perhaps handle the other args in the same way as this so that their order doesn't matter so much... but maybe in a different branch.